File: sort-mapper-example-java/src/main/java/kh/farrukh/sortmapperexamplejava/course/model/CourseResponseDTO.java
package kh.farrukh.sortmapperexamplejava.course.model;

public record CourseResponseDTO(
    Long id,
    String title,
    String description,
    String slogan,
    Long teacherId,
    String teacherFirstName,
    String teacherLastName,
    Integer durationMonth,
    Double price
) {

}

File: sort-mapper-example/src/main/kotlin/kh/farrukh/sortmapperexample/course/model/CourseResponseDTO.kt
package kh.farrukh.sortmapperexample.course.model

data class CourseResponseDTO(
    val id: Long,
    val title: String,
    val description: String,
    val slogan: String?,
    val teacherId: Long,
    val teacherFirstName: String,
    val teacherLastName: String,
    val durationMonth: Int,
    val price: Double
)
